mlrun/serving/states.py

```diff
@@ -1218,12 +1243,90 @@ def __init__(
         self.raise_exception = raise_exception
         self.shape = "folder"
 
+    def add_shared_model_proxy(
+        self,
+        endpoint_name: str,
+        shared_model_name: str,
+        model_artifact: Optional[Union[str, ModelArtifact, LLMPromptArtifact]] = None,
+        labels: Optional[Union[list[str], dict[str, str]]] = None,
+        creation_strategy: Optional[
+            schemas.ModelEndpointCreationStrategy
+        ] = schemas.ModelEndpointCreationStrategy.INPLACE,
```

Comment:
```suggestion
        model_endpoint_creation_strategy: Optional[
            schemas.ModelEndpointCreationStrategy
        ] = schemas.ModelEndpointCreationStrategy.INPLACE,
```
Since this method deals with adding a model, it should be clear that the creation strategy is not for the model, but for the MEP.

Reply:
OK I will change it also on the `add_model` method